services/QuillLMS/client/app/bundles/Staff/components/evidence/syntheticData/seedDataForm.tsx

```diff
@@ -112,11 +135,20 @@ const SeedDataForm = ({ history, match }) => {
 
   function renderExample(value, index, conjunction, exampleIndex) {
     return (
-      <Input
-        handleChange={e => handleExampleChange(e, index, conjunction, exampleIndex)}
-        label={`Example ${exampleIndex + 1}`}
-        value={value}
-      />
+      <div className='example-container'>
+        <Input
+          className="example-input"
+          handleChange={e => handleExampleChange(e, index, conjunction, exampleIndex)}
+          label={`Example ${exampleIndex + 1}`}
+          value={value}
+        />
+        <button
+          className='x-button'
```

Comment:
Note, I tried to get this to work through an intermediary event handler `onRemoveExample` like I did with `onAddExample`, but it didn't work consistently. Some clicks passed `undefined` props to the event handler and some worked as expected, seemingly randomly. I couldn't figure out why, so I called the function directly, which works consistently.

Reply:
Maybe some sort of weird race condition around event cascades?  I think I'm okay with a direct call like this in small cases like this.

Reply:
So `e` was undefined? Weird. I think the approach from the comment above would solve that issue, but I agree with @anathomical that this case is small enough that I don't feel like it's necessary to rewrite it here (the only reason to avoid it is that using an arrow function directly like this creates a new function on every render, which could theoretically break some React optimizations, but I think it would be unnoticeable in virtually every use case and this is an internal tool. also I find them slightly harder to read.)